### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Austinml123**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -84,4 +83,30 @@ <h3>Example Output:</h3>
...
+		li.attr({
+			id: data[i]['episode_id'],
+			date: data[i]['release_date']
+		}).text(data[i]['title']);
+		$('#list').append(li)
+	}
+	$('.starwars').click(function() {
+		alert('onlick is working');
+	});
+  </script>
```
Comment (benjaminboruff) [inline index.html]: Lets start off with a little reorganizing. Move all of the JS into the `js/main.js` file ... so we can work with a smaller file.

Then I think we can get this sorted out.

**Comment by Austinml123** [comment]
Whenever I move the code to js/main.js it does not work for some reason. Even if I move the script tag at the end. Do you know why that would be and how to fix it?

**Comment by benjaminboruff** [comment]
How are you "starting" the ... exercise? Are you just opening the `index.html` file with the browser, or *serving* the file via a webserver like `http-server`? ... the browsing to http://localhost:8080 (or whatever the port is)?

**Comment by Austinml123** [comment]
I am using the localhost:8080 on the http-server like the exercise says on the ReadMe

**Comment by benjaminboruff** [comment]
hmmm, well go ahead and put your code in the `js/main.js` file (and remove it from the `index.html` file) even if it doesn't work, then make a commit, then push those changes up to your repo (the fork), and I'll take a look/download try it myself. It's hard to tell without seeing it :)

**Comment by Austinml123** [comment]
I deleted the script tag from index, but the main.js is now a file by itself and not in the folder. So look at that one and not the one in the folder. Sorry about that. Still getting use to this. There is still a main.js in the folder, but the one that has the code is the one by itself.

**Comment by benjaminboruff** [comment]
Ok, sorry for the delay ... we've been busy :)

There are a couple of ways to make the script start running when the page loads. The easiest is to wrap the function you want to start in `$()`. So, in your case I would wrap you `for` loop and the jQuery `click`: in a function, and start it via the `$()` like this : 
```
$(
		function run() { 
			console.log("Running!!!!!");
		
			for (i = 0; i < data.length; i++) {
				var li = $('<li class = "starwars"></li>');
				li.attr({
					id: data[i]['episode_id'],
					date: data[i]['release_date']
				}).text(data[i]['title']);
				$('#list').append(li)
			}
			$('.starwars').click(function() {
				alert('onlick is working');
			}); 
		}
	)
``` 
You'll see the "Running!!!!!" in the console. Hmmm.... but the DOM isn't getting updated ... I mean the elements are there, but the screen isn't displaying them... that's the next issue!

**Comment by benjaminboruff** [comment]
Oh, wait ... I forgot to refresh the browser .... it works. Man, I've been working with hot-reload servers during development too much. I'm getting rusty! :)

**Comment by benjaminboruff** [comment]
Oh, and you need to change the `inde.html` script tag to link to your `main.js` file ... Looks like your file is in the root, but the tag `<script src="js/main.js"></script>` is linking to the wrong file?

**Pull request closed without merging**